src/image/ImageAnnotator.scss

```diff
@@ -0,0 +1,12 @@
+.bp-image {
+    .ba-Layer {
+        position: absolute;
+        pointer-events: none;
+    }
+
+    // Image element is managed by Preview SDK
+    img.pannable,
```

Comment:
Are these needed? Is there a text layer on our images?

Reply:
Clicking and dragging on top of the image can select it, since images are selectable just like text. This causes some unfortunate visual behavior when drawing out an annotation.

Reply:
I updated the code to only apply this style while the user is drawing. I didn't want to cause any bugs with people not being able to drag-select and copy/paste images.